### DISCUSSION TITLE
How to resume the job by job ID?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [triggerdotdev/trigger.dev#1083] [Q&A] [answered]
I see this option to retry job run from where it brakes. How can I manual stop job and resume job by SDK?
![image](https://github.com/triggerdotdev/trigger.dev/assets/15695252/f96d5e9c-840f-4311-b839-3b52a95436c6)

**Comment by matt-aitken** [comment, marked as answer]
Again sorry the best place for support is [our Discord](https://trigger.dev/discord). In v2 there's no way to resume using the SDK.
